docs/source/guide/index.rst

```diff
@@ -21,8 +21,7 @@ Guide
   :hidden:
   :caption: Explanation
 
-  explanation/algorithms/index
-  explanation/additional_features/index
+  explanation/index
```

Comment:
Why this line is changed?

Reply:
I found that there already exists a parent document with `toctree` for those two contents. It would be cleaner structure to link to the parent document than to link to the child document.

https://github.com/openvinotoolkit/training_extensions/blob/329aead3ae9f714778bfa2d05da1db96d1fb897a/docs/source/guide/explanation/index.rst?plain=1#L20-L25

Reply:
Thanks for explanation